src/main/java/edu/ucsb/cs156/happiercows/controllers/CourseController.java

```diff
@@ -0,0 +1,32 @@
+package edu.ucsb.cs156.happiercows.controllers;
+
+import edu.ucsb.cs156.happiercows.entities.Course;
+import edu.ucsb.cs156.happiercows.repositories.CourseRepository;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import io.swagger.v3.oas.annotations.Operation;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+@Slf4j
+@Tag(name = "Course")
+@RequestMapping("/api/course")
+@RestController
+public class CourseController extends ApiController {
+    @Autowired
+    private CourseRepository courseRepository;
+
+    /**
+     * THis method returns a list of all courses.
```

Comment:
Small capitalization change in comment
```suggestion
     * This method returns a list of all courses.
```